model/geom/barrier.go

```diff
@@ -0,0 +1,6 @@
+package geom
+
+type Barrier struct {
+	StartPoint  Point
+	FinishPoint Point
+}
```

Comment:
сразу надо чтоб если что-то торчит наружу, то сразу интерфейс
```suggestion
package geom

type Barrier interface {
	StartPoint()  Point
	FinishPoint() Point
	setStartPoint()
	setFinishPoint()
}

type barrier struct {
	startPoint  Point
	finishPoint Point
}

// interface method implementation by barrier
```